sample.html

```diff
@@ -1641,12 +1655,12 @@ <h4 class="panel-title">Options</h4>
     function sendSerialData() {
         var properties = {
             baudRate: $("#serialBaud").val(),
+            type: $("input[name='serialType']:checked").val(),
```

Comment:
As discussed with @bberenz, this should be part of the data object, not the connection properties.

Reply:
on it